From 4e9334e4daa0adc5f11efa97a2d70129ea384132 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Tue, 17 Jul 2018 13:23:51 +0200
Subject: [PATCH] curl: Disable libpsl

This library is available in packages repository, but not in the main one, so
sometimes it can get autodetected and added as dependency.

Signed-off-by: Michal Hrusecky <michal.hrusecky@nic.cz>
---
 package/network/utils/curl/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/network/utils/curl/Makefile b/package/network/utils/curl/Makefile
index 2e1e909..2f7cc7e 100644
--- a/package/network/utils/curl/Makefile
+++ b/package/network/utils/curl/Makefile
@@ -113,6 +113,7 @@ CONFIGURE_ARGS += \
 	--without-libmetalink \
 	--without-librtmp \
 	--without-libidn \
+	--without-libpsl \
 	--without-ca-path \
 	--with-ca-bundle=/etc/ssl/certs/ca-certificates.crt \
 	\
-- 
2.18.0

